src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -76,7 +95,19 @@
 	}
 
 	public RobotMap() {
+		Component.joystick = new CustomJoystick(Port.HumanInput.joystick);
 		Component.pdp = new PDP();
+		Component.crateIORollerLeft = new Motor("CrateIORollers", new CANTalonSRX(Port.CANMotor.crateIORollerMotorLeft));
+		Component.crateIORollerRight = new Motor("CrateIORollers", new CANTalonSRX(Port.CANMotor.crateIORollerMotorRight));
+		Component.crateIO = new CrateIO(Component.crateIORollerLeft, Component.crateIORollerRight);
+		Component.rollyBOIRollerLeft = new Motor("RollyBOIRollers", new CANTalonSRX(Port.CANMotor.rollyBOIRollerMotorLeft));
+		Component.rollyBOIRollerLeft.setInverted(true);
+		Component.rollyBOIRollerRight = new Motor("RollyBOIRollers", new CANTalonSRX(Port.CANMotor.rollyBOIRollerMotorRight));
```

Comment:
My God I can't believe I forgot about this... Motors can take in multiple speed controllers, meaning we may be able to reduce these two 1 motor. I'm not saying we should, or that it's the correct implemetnation, but look at how the left and right motors are defined

Reply:
Don't think this will work because one of the motors is inverted, and the other one isn't